### DISCUSSION TITLE
How to add module with join's

### DISCUSSION THREAD

**Discussion by rscarpim1973** [apostrophecms/apostrophe#3727] [General]
Hello everyone.
I just would like to know how may I create a new module with joins.

Example:

I have a news post, that should have a one-to-many relation called related news, I just would like to know using apostrophe v2 how to achieve that using the command line to create, I tried hardcoding but I got an error message when I click at the Browse button
<img width="723" alt="Screen Shot 2022-04-22 at 10 22 53 AM" src="https://user-images.githubusercontent.com/104212714/164734126-0580e7df-ff7c-4f02-9a63-cc7189a48b4d.png">

**Comment by boutell** [comment]
Hi @rscarpim1973, there is no CLI for creating a join between two types. Adding schema fields, like joins, is done in your code.

There is a really good explanation here with examples:

https://v2.docs.apostrophecms.org/core-concepts/reusable-content-pieces/joins.html

If you have any specific followup questions after following this guide please share the details here.

**Comment by rscarpim1973** [comment]
Thank you very much for you help boutell, I did try that code using the article above, but the way that they set up the project it's not allowing me, because I can't see the new item created, here's how they structure the application:
<img width="411" alt="Screen Shot 2022-04-25 at 9 06 33 AM" src="https://user-images.githubusercontent.com/104212714/165095499-acdd05c2-8a61-4cd6-996e-8db848ce747c.png">


Please advise, if I need to change any file, I did try to change the app.js but still not seeing.

Thank you very much.

**Reply by boutell** [reply]
This is a multisite project. Please see the multisite documentation:

https://github.com/apostrophecms/apostrophe-multisite

We also encourage developers using multisite to reach out to us for enterprise support:

https://apostrophecms.com/contact-us

**Comment by rscarpim1973** [comment]
Thank you so much for the help, It did helped me out to implement the one-to-many relationship, and its working!!
Now as usual I have another question,lol

How can I add a new tab for example to customize the css file here:
<img width="435" alt="Screen Shot 2022-04-26 at 4 15 52 PM" src="https://user-images.githubusercontent.com/104212714/165386568-affcebe8-d5c5-42be-93ad-a42586883c98.png">

Thank you so much for once more helping me out.
